src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
 export interface Figure {
-
+  shape: string;
```

Comment:
you can specify this type more precisely
```
type Shape = 'triangle' | 'circle' | 'rectangle'
```

```suggestion
  shape: Shape;
```